tools/tests/test_roster.py

```
@@ -706,4 +748,253 @@ def test_a_crlf_cell_with_genuine_drift_still_reports(tmp_path):
+    A containment check comparing the resolved surface against the *unresolved*
+    root reds every entry on a repository that itself lives under a link --
+    which is an ordinary way to reach a checkout and has nothing to do with the
+    defect. Comparing against `root.resolve() / directory` passes it, because
+    on such a tree the surface genuinely resolves elsewhere and is still where
+    it was declared to be. A guard that blocks lawful work fails as hard as one
+    that passes unlawful work. [PR #278 review, F1]
+    """
+    make_cell(tmp_path, "alpha")
+    real = tmp_path / "real"
+    real.mkdir()
+    for surface in roster.SURFACES:
```

Ruff (0.16.2)

[warning] Loop control variable `surface` not used within loop body

(B007)